src/astronomy/moshier/aberration.js

```diff
@@ -1,6 +1,6 @@
 $ns.aberration = {}
 
-$ns.aberration.calc = function (p, result) {
+$ns.aberration.calc = function (p) {
```

Comment:
Similar recursion confirmation check needed

Reply:
Yes, I checked this.... there are many "calc" functions, so the string search for ".calc" yields 115 results in 25 files. While scanning these results to check for usages of the aberration calc function is still manageable, the search can be narrowed to search for the string "aberration.calc" but then one should also search for "this.calc" to make sure no recursive calls are calls are missed. Anyway, I've checked this. I feel safe about making this change but also encourage you to verify. I think this method of searching for the function name through all the source files is rather quick and easy to do as a means of verification.